css/style.css

```
@@ -0,0 +1,153 @@
+/*=============================================================================
+ベース
+==============================================================================*/
+*, *::before, *::after {
+  box-sizing:inherit;
+}
+
+html {
+  box-sizing:border-box;
+}
+
+body {
+  color:#333;
+  font-family:'Roboto','sans-serif';
+  line-height:1.8;
+  font-size:14px;
+}
+
+img {
+  width:100%;
+  height:auto;
+  vertical-align:bottom;
+}
+
+a {
+  text-decoration:none;
+  color:inherit;
+}
+
+a:hover {
+  opacity:0.7;
+}
+/*=============================================================================
+グローバル
+==============================================================================*/
+.btn {
+  display:flex;
+  justify-content:center;
+  align-items:center;
+  background-color:#ffa800;
+  color:#fff;
+  border-radius:9999px;
+  border-style:none;
+  cursor:pointer;
+  font-weight:bold;
+}
+
+.pcNone {
+  display:none;
+}
+
+/*============================================================================*/
+header {
+  background-color:#fff;
+  position:sticky;
+  top:0;
+  box-shadow:rgba(0, 0, 0, 0.3) 1px 1px 3px;
+}
+
+header .inner {
+  max-width:980px;
+  width:96%;
+  height:60px;
+  display:flex;
+  justify-content:space-between;
+  align-items:center;
+  margin:auto;
+}
+
+header .logo a img {
+  display:flex;
+  width:100px;
+  height:20px;
+}
```

```suggestion
header .logo a {
  display:flex;
}
header .logo img {
  width:100px;
  height:20px;
}
```
imgタグの指定は、幅と高さだけにしたほうがいいです。

---

> 

修正しました。
1f9a0fa1435a2985eddd8c9a1be352080798165f